pytest.ini

```diff
@@ -0,0 +1,7 @@
+[pytest]
+addopts =
+    --cov
```

Comment:
This line will be picked up by `pytest`, so it will invoke the `pytest-cov` plug-in (added to the `requirements/tests.txt` file).